client/src/routes/admin/AdminPage.tsx

```diff
@@ -14,9 +15,55 @@ import React from 'react';
 import UserContext from '../../context/UserContext';
 import img from '../../assets/poster.png';
 import API from '../../api/apiRoutes';
+import Searchbar from '../../components/Searchbar';
+import { 
+	useNavigate 
+} from 'react-router-dom';
+import {
+	identity, map 
+} from 'cypress/types/lodash';
 
 function AdminPage() : JSX.Element{
 	const { theme } = React.useContext(UserContext);
+	const navigate = useNavigate();
+	
+	const getAllUsers = () => {
+		const result: string[] = [];
+		API('gebruikers').GetAll().then((res)=>{
+			res.data.forEach((e: { naam: any; }) => {
+				result.push(e.naam);
+			});
+		}).catch((error) => {
+			console.log(error);
+		});
+
+		// const res = await API('gebruikers').GetAll();
+		// console.log(res.data);
+		// const result: string[] = [];
+		// res.data.forEach((e: { naam: string; }) => {
+		// 	result.push(e.naam);
+		// });
+		return result;
+	};
+
+	const getIdFromName = async(name: any) => {
+		let id = 0;
+
+		const response = await API('gebruikers').GetAll();
```

Comment:
If we already have an array with all the users loaded why query it again? kinda unnecessary.